src/pathfinding_service/claim_fees.py

```diff
@@ -9,15 +9,15 @@
 from pathfinding_service.database import PFSDatabase
 from pathfinding_service.model import IOU
 from raiden.utils.typing import BlockNumber, TokenAmount
-from raiden_contracts.constants import CONTRACT_ONE_TO_N, CONTRACT_USER_DEPOSIT
+from raiden_contracts.constants import CONTRACT_ONE_TO_N
 from raiden_contracts.contract_manager import ContractManager, contracts_precompiled_path
 from raiden_libs.cli import blockchain_options, common_options
 from raiden_libs.utils import private_key_to_address
 
 log = structlog.get_logger(__name__)
 
 
-@blockchain_options()
+@blockchain_options(contracts=[CONTRACT_ONE_TO_N])
```

Comment:
Don't we need the user deposit contract as well? Otherwise we should check the deposit before sending the transaction. Could you open a ticket in that case?

Reply:
The UDC is used by the OneToN contract. There's no need to use it directly, since the claim function  returns the actually claimable amount. That's the reason why we do a `call` first only when the returned value is large enough a `transact`.